Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -2634,35 +2645,26 @@ namespace AssetProcessor
             // examination occurs here.
             // first, is it a source or is it a product in the cache folder?
             QString normalizedPath = examineFile.m_fileName;
-
-            AZ_TracePrintf(AssetProcessor::DebugChannel, "ProcessFilesToExamineQueue: %s delete: %s.\n", examineFile.m_fileName.toUtf8().constData(), examineFile.m_isDelete ? "true" : "false");
-
-            // debug-only check to make sure our assumption about normalization is correct.
-            Q_ASSERT(normalizedPath == AssetUtilities::NormalizeFilePath(normalizedPath));
-
-            // if its in the cache root then its a product file:
-            bool isProductFile = IsInCacheFolder(examineFile.m_fileName.toUtf8().constData());
-#if AZ_TRAIT_OS_PLATFORM_APPLE
-            // a case can occur on apple platforms in the temp folders
-            // where there is a symlink and /var/folders/.../ is also known
-            // as just /private/var/folders/...
-            // this tends to happen for delete notifies and we can't canonicalize incoming delete notifies
-            // because the file has already been deleted and thus its canonical path cannot be found.  Instead
-            // we will use the canonical path of the cache root dir instead, and then alter the file
-            // to have the current cache root dir instead.
-            if ((!isProductFile)&&(!canonicalRootDir.isEmpty()))
-            {
-                // try the canonicalized form:
-                isProductFile = examineFile.m_fileName.startsWith(canonicalRootDir);
-                if (isProductFile)
+            
+            if (needCanonicalFixup)
+            {
+                if (normalizedPath.startsWith(canonicalRootDir))
                 {
                     // found in canonical location, update its normalized path
                     QString withoutCachePath = normalizedPath.mid(canonicalRootDir.length() + 1);
                     // the extra +1 is to consume the slash that is after the root dir.
                     normalizedPath = AssetUtilities::NormalizeFilePath(m_cacheRootDir.absoluteFilePath(withoutCachePath));
                 }
             }
-#endif // AZ_TRAIT_OS_PLATFORM_APPLE
+
+            AZ_TracePrintf(AssetProcessor::DebugChannel, "ProcessFilesToExamineQueue: %s delete: %s.\n", examineFile.m_fileName.toUtf8().constData(), examineFile.m_isDelete ? "true" : "false");
```

Comment:
Nit: To me its unclear what the printf is trying to say, I believe its more that "ProcessFilesToExamineQueue: File %s is marked for delete %s" 

* Not sure why spamming about all files here is useful esp as we are not currently deleting, or removing files.

Reply:
Note that I didn't add this, it was present earlier.  GITHUB diff seems to think I deleted it and added it instead of just moved it around.

I think this is part of the scrub done a couple years ago so that the log file contains the absolute minimal amount of information to know what branches were taken.   For example you'll see no other output anywhere else that indicates that this file made it to the final analysis queue except if something goes wrong.

I'm hesitant to remove this... really, if we do, it should be part of a much bigger scrub that moves messages like this into some "debug trace" severity and turns them off by default unless you change a checkbox or config somewhere...